deployment/main.tf

```diff
@@ -0,0 +1,116 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+resource "aws_ecs_cluster" "this" {
+  name = "akka-app-cluster"
+  tags = {
+    Project = "aipcc-akka"
+  }
+}
+
+resource "aws_iam_role" "ecs_task_execution" {
+  name = "ecsTaskExecutionRole"
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17",
+    Statement = [{
+      Effect = "Allow",
+      Principal = {
+        Service = "ecs-tasks.amazonaws.com"
+      },
+      Action = "sts:AssumeRole"
+    }]
+  })
+  tags = {
```

Comment:
Can we set the tags once and have them automatically applied to all resources, instead of repeating them everywhere?

Reply:
Yes, added it as local variable